deploy/sre-1270-rebalancing-infra-nodes/00-sre-1270-rebalancing-infra-nodes.ServiceAccount.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sre-1270-rebalancing-infra-nodes
```

Comment:
what does 1270 means?

Reply:
1270 is the epic number.

Reply:
Thank you again for review. I have updated to `osd-rebalance-infra-nodes`.